muikku-atests/src/test/java/fi/muikku/ui/AbstractUITest.java

```diff
@@ -261,6 +269,11 @@ public Boolean apply(WebDriver driver) {
     });
   }
 
+  protected void waitForPresentVisible(String selector) {
```

Comment:
waitForPresentAssertVisible would be more descriptive name, waitForPresentVisible is confusing